src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -428,7 +428,7 @@ def run_container(container_name, image, flags=None, command=""):
 
 def remove_running_container(container_name):
     log.info(f"Removing Container {container_name}")
-    container_rm_cmd = f"podman-remote stop {container_name} && podman-remote rm {container_name}"
+    container_rm_cmd = f"podman-remote kill {container_name}"
```

Comment:
wait, this is odd
we don't need to remove the container? (it's also in the function name and in the log, so we shouldn't do something different than that)

Reply:
> wait, this is odd we don't need to remove the container? (it's also in the function name and in the log, so we shouldn't do something different than that)

From the trace in CI run , here is the fail point

 yield start_proxy_server
        if global_variables.test_teardown:
            log.info("--- TEARDOWN --- proxy controller")
            for server in proxy_servers:
>               server.remove()

src/tests/base_test.py:724: 

https://ci-jenkins-csb-kniqe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-assisted-installer-virt-tf/5136//testReport/junit/test_proxy/TestProxy/Run_assisted_installer_Functional_API_Tests___test_http_proxy_http_proxy_params1_https_proxy_params1_OCP_35314_/

Reply:
yeah but my argument here is that this code is misleading. method says ``remove_running_container`` and we're doing another thing

regardless, the error says ``cannot remove container ... as it is running - running or paused containers cannot be removed without force`` so I think we should use the force :smile: (going back to my suggestion to just do ``podman-remote rm --force ...`` and close the dill)

Reply:
> yeah but my argument here is that this code is misleading. method says `remove_running_container` and we're doing another thing
> 
> regardless, the error says `cannot remove container ... as it is running - running or paused containers cannot be removed without force` so I think we should use the force smile (going back to my suggestion to just do `podman-remote rm --force ...` and close the dill)
sure no problem....